src/main/java/team9/ddang/dog/controller/DogController.java

```diff
@@ -0,0 +1,46 @@
+package team9.ddang.dog.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import team9.ddang.dog.dto.CreateDogRequest;
+import team9.ddang.dog.dto.UpdateDogRequest;
+import team9.ddang.dog.service.DogService;
+
+@RestController
+@RequestMapping("/dogs")
+@RequiredArgsConstructor
+public class DogController {
+
+    private final DogService dogService;
+
+    @PostMapping
+    public ResponseEntity<Void> createDog(@RequestBody CreateDogRequest request) {
+        // Service 계층 호출
+        dogService.createDog(request.toServiceRequest());
+        return ResponseEntity.status(HttpStatus.CREATED).build();
```

Comment:
```suggestion
    public ApiResponse<CreateDogResponse> createDog(@RequestBody CreateDogRequest request) {
        // Service 계층 호출
        dogService.createDog(request.toServiceRequest());
        return ApiResponse.created(CreateDogResponse);
```

해당 부분은 저희가 공통응답을 사용하고 있습니다! 또 data 에는 create 의 경우 어떤 개가 만들어졌는지 간단하게 띄워주는 게 좋을 것 같아요